docs/howto_guides/Realtime-Best-Practices.md

```diff
@@ -0,0 +1,497 @@
+# Trick Real-time Best Practices
+
+**Contents**
+
+* [Purpose](#Purpose)<br>
+* [Prerequisite Knowledge](#prerequisite-knowledge)<br>
+* [Do's, Don'ts and Wisdom](#guidelines)<br>
+
+<a id= introduction></a>
+
+---
+## Purpose
+The intention of this document is to compile and share practical knowledge, based on the experience of people in the Trick simulation community regarding the development of real-time computer simulations.
+
+<a id=prerequisite-knowledge></a>
+## Prerequisite Knowledge
+(Assuming you've completed the [Trick Tutorial] (https://nasa.github.io/trick/tutorial/Tutorial))
+
+--
+
+<a id=simulation-time-vs-real-time></a>
+### Simulation Time vs Real Time
+
+Real world dynamic systems change in real time (the time that you and I experience). A simulated dynamic system changes in simulation-time. Simulation time begins at t=0, and runs until we stop it. Simulation time can elapse faster or slower than real-time.
+
+If we want to interact with a simulation as if it were real, we need to synchronize simulation time to real time. This requires that a simulation is capable of running at least as fast as real time. If it is incapable, then it can not be made to run in real time.

```

Comment:
Do we want to use either real-time or real time consistently?

Reply:
Same as above issue. Fixed.